test/e2e/network/proxy.go

```diff
@@ -58,6 +62,8 @@ const (
 	proxyHTTPCallTimeout = 30 * time.Second
 	podRetryPeriod       = 1 * time.Second
 	podRetryTimeout      = 1 * time.Minute
+
+	svcReadyTimeout = 1 * time.Minute
```

Comment:
How about using https://github.com/kubernetes/kubernetes/blob/f5bb2c19b9b6a4138e67c6da82e477e50a1ce5ae/test/e2e/framework/util.go#L109 instead?

Reply:
Fixed.

@oomichi Is there anything else before this can be approved / lgtm ?

Reply:
@heyste Why don't we use framework.ServicesStartTime instead of defining the same value?

Reply:
Sorry, miss read the request before. I've just noticed that those constants listed in [framework/util.go](https://github.com/kubernetes/kubernetes/blob/f5bb2c19b9b6a4138e67c6da82e477e50a1ce5ae/test/e2e/framework/util.go#L86) have a deprecated notice above them.

Is there another timeout I should be using?

Reply:
@heyste
Thanks for pointing it out here.
TBH I never know that these constraints are deprecated since https://github.com/kubernetes/kubernetes/pull/96042
It is fine to keep current way here.